src/parser/contexts.h

```diff
@@ -1759,7 +1764,8 @@ struct ParseDefsCtx : TypeParserCtx<ParseDefsCtx> {
       assert(wasm.debugInfoFileNames.size() == it->second);
       wasm.debugInfoFileNames.push_back(std::string(file));
     }
-    irBuilder.setDebugLocation({it->second, *line, *col});
+    irBuilder.setDebugLocation(
+      Function::DebugLocation({it->second, *line, *col}));
```

Comment:
Why is this needed?

Reply:
`setDebugLocation` has been changed to take a `std::optional<Function::DebugLocation>`, so the previous code no longer compiles.